pam/integration-tests/cli_test.go

```diff
@@ -39,10 +40,12 @@ func TestCLIAuthenticate(t *testing.T) {
 		"Authenticate user with form mode with button":                {tape: "form_with_button"},
 		"Authenticate user with qr code":                              {tape: "qr_code"},
 		"Authenticate user and reset password while enforcing policy": {tape: "mandatory_password_reset"},
-		"Authenticate user and offer password reset":                  {tape: "optional_password_reset"},
+		"Authenticate user and reset password with mismatch":          {tape: "mandatory_password_reset_with_mismatch"},
```

Comment:
A question: now that you rebased on main, this is already covered by another test case, no?

Reply:
It is in the `passwd` case, not in the `login` one, so even if they go through the same code-paths, I wanted to ensure it worked in both cases.

Reply:
Sure, but we still have it in `Deny authentication if password does not match the required criteria`

Reply:
Oh, yeah... I think that's fine, my test wanted to check only the mismatch case, but it's covered there now.

Reply:
I don't see any issue in the golden files